### PR TITLE
textures: dds parser

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

**Review by dryabinin-actionengine — APPROVED** [review]

File: modules/basis/src/lib/parsers/parse-pvr.js [hunk 1]
```diff
@@ -3,8 +3,9 @@
 // Copyright (c) 2017 Tarek Sherif, The MIT License (MIT)
 import {GL} from '../gl-constants';
 
-const PVR_CONSTANTS = {
+export const PVR_CONSTANTS = {
   MAGIC_NUMBER: 0x03525650,
+  MAGIC_NUMBER_EXTRA: 0x50565203,
```
Comment (ibgreen) [inline modules/basis/src/lib/parsers/parse-pvr.js:8]: Can we have better names? How are these different? Any links?

File: modules/basis/src/lib/parsers/parse-dds.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import {assert} from '@loaders.gl/core/';
```
Comment (ibgreen) [inline modules/basis/src/lib/parsers/parse-dds.js:1]: Was this copied from some open source code? Should we add some header?

File: modules/basis/src/lib/parsers/parse-compressed-texture.js [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
+  const version = header[PVR_CONSTANTS.MAGIC_NUMBER_INDEX];
 
-  if (version === 0x03525650 || version === 0x50565203) {
-    return true;
-  }
-  return false;
+  return version === PVR_CONSTANTS.MAGIC_NUMBER || version === PVR_CONSTANTS.MAGIC_NUMBER_EXTRA;
+}
+
+function isDds(data) {
```
Comment (ibgreen) [inline modules/basis/src/lib/parsers/parse-compressed-texture.js:20]: Formats are all caps? `isPVR`, `isDDS` ?

**Review by ibgreen — APPROVED** [review]

File: modules/basis/src/lib/parsers/parse-pvr.js (hunk 1 above)
Comment (belom88) [reply modules/basis/src/lib/parsers/parse-pvr.js:8]: For DDS this byte is called "VERSION" there are two possible versions. This information is described in spec: 
http://cdn.imgtec.com/sdk-documentation/PVR+File+Format.Specification.pdf

File: modules/basis/src/lib/parsers/parse-dds.js (hunk 2 above)
Comment (belom88) [reply modules/basis/src/lib/parsers/parse-dds.js:1]: I don't see your point here.

**Pull request merged**